frontend/javascripts/oxalis/controller/segment_mesh_controller.ts

```diff
@@ -182,9 +182,7 @@ export default class SegmentMeshController {
   ): void {
     const additionalCoordinatesString = getAdditionalCoordinatesAsString(additionalCoordinates);
     const keys = [additionalCoordinatesString, layerName, segmentId, lod];
-    const isNewlyAddedMesh =
-      this.meshesGroupsPerSegmentId[additionalCoordinatesString]?.[layerName]?.[segmentId]?.[lod] ==
-      null;
+    const isNewlyAddedMesh = _.get(this.meshesGroupsPerSegmentId, keys) == null;
```

Comment:
This is just a refactoring, correct? Or was there a bug here?

Reply:
it's refactoring